### DISCUSSION TITLE
Minor change in help docs for creating katas

### DISCUSSION THREAD

**Discussion by stonecrusher** [codewars/docs#351] [Ideas]
Hi,
I just wanted to create my first kata and I came across this little naming issue:

Help docs say `correct solution block` but in fact it is named `complete solution`.
Help docs say `solution setup` but in fact it is named `initial solution`.

Also weird line breaks e.g.  
after `The preloaded code block is optional and you do` or  
after `This allows you to pick and choose what a user should and shouldn't see. Don't forget`

Hopefully I will manage to get the test framework setup right now :-)

**Comment by hobovsky** [comment]
Codewars documentations has its own repository on Github: https://github.com/codewars/docs/issues
I will move this issue there.

Indeed, naming of snippets is inconsistent between UI, users, and documentation. There are more occurrences of such inconsistencies.

The docs repo is open for contributions, so if you have any specific proposals for fixing, feel free to submit a pull request.
